packages/suite/src/views/wallet/receive/components/CoinjoinReceiveWarning.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { darken } from 'polished';
+import { useDispatch, useSelector } from '@suite-hooks';
+import { FiatValue, FormattedCryptoAmount, Translation } from '@suite-components';
+import { Button, Icon, variables, Warning } from '@trezor/components';
+import { hideCoinjoinReceiveWarning } from '@suite-actions/suiteActions';
+import { formatAmount, getAccountDecimals } from '@suite-common/wallet-utils';
+import { UNECONOMICAL_COINJOIN_THRESHOLD } from '@suite/services/coinjoin';
+import { selectSelectedAccount } from '@wallet-reducers/selectedAccountReducer';
+import { selectIsAccountWithRatesByKey } from '@suite-common/wallet-core';
+
+const StyledWarning = styled(Warning)`
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        display: grid;
+        grid-template-columns: 1fr auto;
+    }
+`;
+
+const InfoIcon = styled(Icon)`
+    width: 18px;
+    height: 18px;
+    margin-right: 6px;
+    align-self: flex-start;
+    background: ${({ theme }) => theme.TYPE_LIGHT_ORANGE};
+    border-radius: 50%;
+
+    ${variables.SCREEN_QUERY.MOBILE} {
+        align-self: start;
+    }
+`;
+
+const Text = styled.div`
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const Heading = styled.p`
```

Comment:
```suggestion
const Heading = styled.div`
```

See console warning: `validateDOMNesting(...): <div> cannot appear as a descendant of <p>`

Reply:
0f15f72eb